docs/app/pages/components/components-sections/color/color-picker/color-picker.component.html

```diff
@@ -1,53 +1,42 @@
 <div class="demo-color-picker-container">
-    <div class="btn-group"
-        (keydown.esc)="isPickerOpen = false; toggleButton.focus()"
-        dropdown
-        [(isOpen)]="isPickerOpen"
-        [autoClose]="false">
-
-        <button type="button"
-            aria-label="Select a color"
-            class="btn button-secondary dropdown-toggle"
-            #toggleButton
-            dropdownToggle>
-
-            <span class="ux-color-picker-button-preview" [style.background-color]="selected.rgba">
-                &nbsp;
-            </span>
-            <ux-icon name="chevron-down"></ux-icon>
-
-        </button>
-
-        <div *dropdownMenu
-            #dropdownMenu
-            id="color-picker-dropdown"
-            class="dropdown-menu ux-color-picker-dropdown">
-
-            <ux-color-picker
-                [colors]="colors"
-                [(selected)]="selected"
-                (selectedChange)="colorPickerSelectedChange(); toggleButton.focus()"
-                [columns]="columns"
-                [buttonStyle]="buttonStyle"
-                [buttonSize]="buttonSize"
-                [showTooltips]="showTooltips"
-                [showInput]="showInput"
-                (inputSubmit)="isPickerOpen = false"></ux-color-picker>
-
-            <div *ngIf="showInput" class="btn-container demo-color-picker-close">
-                <button type="button" class="btn button-primary" (click)="isPickerOpen = false">
-                    Accept
-                </button>
-            </div>
-
-        </div>
-    </div>
+    <button
+        type="button"
+        aria-label="Select a color"
+        class="btn button-secondary dropdown-toggle"
+        #toggleButton
+        [uxMenuTriggerFor]="dropdownMenu"
+    >
+        <span class="ux-color-picker-button-preview" [style.background-color]="selected.rgba">
+            &nbsp;
+        </span>
+        <ux-icon name="chevron-down"></ux-icon>
+    </button>
 
     <span class="m-l-lg">
-        Selected: <code>{{selected.hex}}</code> &mdash; <code>{{selected.rgba}}</code>
+        Selected: <code>{{ selected.hex }}</code> &mdash;
+        <code>{{ selected.rgba }}</code>
     </span>
 </div>
 
+<ux-menu #dropdownMenu id="color-picker-dropdown">
+    <div uxMenuItem [closeOnSelect]="false">
```

Comment:
I tried using this but in the content of the application there are some issues with tabbing into the dropdown. It seems if there are other elements in the tabindex it will prioritise those over the dropdown - https://codesandbox.io/s/color-picker-ux-aspects-forked-8fdz3l?file=/src/app/app.component.html

Reply:
Sure, but this is a hack - you can see that the background color changes, and the pointer cursor is active on the entire picker. If you're going to use ux-menu here then it needs to be an adequate replacement without degrading the user experience. Maybe you need to modify ux-menu so that it works properly with custom content as well as with menu items, for example.

Reply:
Agreed, I had a play around with this using the `uxMenuTabbableItem` directive. It appears to work quite well as it can be applied to any element. The only thing about it is the extra tab step but it doesn't have the background or clickable background issue when using the `uxMenuItem` directive. 

https://codesandbox.io/s/color-picker-ux-aspects-forked-8fdz3l?file=/src/app/app.component.html

Just thought I should get an opinion on this before making any changes to the ux-menu.